int32 drone_id
time start_time

float64 dt
geometry_msgs/Point[] pos_pts
float64[] yaw_pts
